### PR TITLE
Add Polish translations for Jenkins plugin manager

### PR DESCRIPTION
**Pull request by damianszczepanik**
### Testing done
Tested locally on version 2.385

### Submitter checklist
- [ ] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.

### Desired reviewers

@jenkinsci/core-pr-reviewers

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7593"></a>

### CODE REVIEW

File: core/src/main/resources/hudson/PluginManager/sidepanel_pl.properties
Comment (basil) [inline core/src/main/resources/hudson/PluginManager/sidepanel_pl.properties:3]: This warms my heart (my Sun Microsystems offer letter from 2007 remains a cherished piece of tech memorabilia), but sadly Sun Microsystems ceased to exist as a legal entity about 13 years ago. For new files, the copyright should be assigned solely to you:

```suggestion
# Copyright (c) 2023, Damian Szczepanik
```

File: core/src/main/resources/lib/layout/search-bar_pl.properties
Comment (basil) [inline core/src/main/resources/lib/layout/search-bar_pl.properties:3]: Ditto:

```suggestion
# Copyright (c) 2023, Damian Szczepanik
```

**Review by basil — DISMISSED** [review]

File: core/src/main/resources/lib/layout/search-bar_pl.properties
Comment (basil) [reply core/src/main/resources/lib/layout/search-bar_pl.properties:3]: Unresolving. This still seems to read "Sun Microsystems" in the latest version of this PR.

**Comment by damianszczepanik** [comment]
Should be fine now. I guess changes were rollback after operation "branch update" on github

File: core/src/main/resources/hudson/PluginManager/advanced_pl.properties
Comment (basil) [inline core/src/main/resources/hudson/PluginManager/advanced_pl.properties:22]: `core/src/main/resources/hudson/PluginManager/advanced.jelly` no longer contains an "Advanced Settings" with a capital S after #6783, so this line can be deleted:

```suggestion
```

File: core/src/main/resources/hudson/PluginManager/available_pl.properties
Comment (basil) [inline core/src/main/resources/hudson/PluginManager/available_pl.properties:22]: `core/src/main/resources/hudson/PluginManager/available.jelly` no longer contains an "Available Plugins" string with a capital P, so this line can be deleted:

```suggestion
```

File: core/src/main/resources/hudson/PluginManager/installed_pl.properties
Comment (basil) [inline core/src/main/resources/hudson/PluginManager/installed_pl.properties:22]: `core/src/main/resources/hudson/PluginManager/installed.jelly` no longer contains an "Installed Plugins" string with a capital I, so this line can be deleted:

```suggestion
```

File: core/src/main/resources/hudson/PluginManager/table_pl.properties
Comment (basil) [inline core/src/main/resources/hudson/PluginManager/table_pl.properties:26]: This key has been deleted since #4534:

```suggestion
```

**Review by basil — APPROVED** [review]
Looks good to me. Left some optional cleanup suggestions for your consideration.

File: core/src/main/resources/hudson/PluginManager/advanced_pl.properties
Comment (damianszczepanik) [reply core/src/main/resources/hudson/PluginManager/advanced_pl.properties:22]: Then this is technical debt of mentioned PR and should be fixed as follow-up of them. Not by removing Polish translation but removing items from all localization.

Comment (basil) [reply core/src/main/resources/hudson/PluginManager/advanced_pl.properties:22]: Indeed. For this reason, these suggestions were merely optional.

**Review by timja — APPROVED** [review]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Review by NotMyFault — APPROVED** [review]
Thanks!

**Pull request merged**
